tools/user-report-web/main.go

```diff
@@ -39,6 +39,7 @@ var (
 	timeoutFlag = flag.Duration("timeout", 5*time.Second, "request time out duration in the format: 0h0m0s")
 	apikeyFlag  = flag.String("apikey", "", "API Key to use")
 	addrFlag    = flag.String("addr", "http://localhost:8080", "protocol, address and port on which to make the API call")
+	sendGetFlag = flag.Bool("send-get", false, "If true, will use GET instead of POST to initiate the request")
```

Comment:
Nit and can be follow-up. It'd probably be better to have the flag be http-method and accept that as a param on the client (instead of a bool)